source/admin_manual/lua.rst

```diff
@@ -108,6 +108,8 @@ Lua API
 
 .. py:function:: client({timeout=milliseconds, max_attempts=number, debug=boolean})
 
+   .. versionadded:: 2.3.19
+
```

Comment:
With all the other Lua functions the versionadded is the last element, not the first. Should keep consistent.